### DISCUSSION TITLE
how to unlock a character by consecutive wins

### DISCUSSION THREAD

**Discussion by NenelFX** [ikemen-engine/Ikemen-GO#1782] [Q&A]
I would like to unlock the character through consecutive wins

unlock = stats.modes ~= nil and stats.modes.arcade ~= nil and stats.modes.arcade.clear >= 1
and use consecutive wins

and I would also like to know if you can use "arcade.clearcount" as a "trigger"?

**Comment by senatormailman** [comment]
I think having a loss count would be good as well for this. It would really be fun to have like a "pity streak" as an unlock feature in vs/netplay.
